test_cases/stdlib/check_contextlib.py

```diff
@@ -18,3 +19,17 @@ class Thing(ExitStack):
     assert_type(cm, ExitStack)
 with thing as cm2:
     assert_type(cm2, Thing)
+
+
+@asynccontextmanager
+async def async_context() -> AsyncGenerator[str, None]:
+    yield "example"
+
+
+async def async_gen() -> AsyncGenerator[str, None]:
+    yield "async gen"
+
+
+@asynccontextmanager
+def async_cm_func() -> AsyncGenerator[str, None]:
+    return async_gen()
```

Comment:
All these test cases appear to pass on `main`

Reply:
Yes, mypy already knows how to solve this on `main` 👍

Reply:
> Yes, mypy already knows how to solve this on `main` 👍

Then I'm confused. What's the problem we're trying to fix, if these tests all already pass?

Reply:
Yeap, since it was fixed on `main`, we can drop this fix from `typeshed`